lib/engine/game/g_18_norway/map.rb

```diff
@@ -0,0 +1,327 @@
+        }.freeze
+
+        TILES = {
+          '1' => 1,
+          '2' => 1,
+          '3' => 3,
+          '4' => 3,
+          '5' => 4,
+          '6' => 4,
+          '7' => 3,
+          '8' => 12,
+          '9' => 6,
+          '55' => 1,
+          '56' => 1,
+          '57' => 4,
+          '58' => 3,
+          '69' => 1,
+
+          '16' => 1,
+          '18' => 1,
+          '19' => 1,
+          '20' => 1,
+          '23' => 3,
+          '24' => 3,
+          '25' => 3,
+          '26' => 2,
+          '27' => 2,
+          '28' => 2,
+          '29' => 2,
+          '87' => 1,
+          '88' => 1,
+          '204' => 1,
+          '441' => 2,
+          '442' => 2,
+          '443' => 1,
+          '444' => 2,
+          'O1' => {
+            'count' => 1,
+            'color' => 'green',
+            'code' => 'label=O;city=revenue:60;city=revenue:60;city=revenue:60;path=a:0,b:_0;'\
+                      'path=a:3,b:_0;path=a:2,b:_1;path=a:5,b:_1;path=a:1,b:_2;path=a:4,b:_2',
+          },
+          'S1' => {
+            'count' => 1,
+            'color' => 'green',
+            'code' => 'label=S;city=revenue:30;path=a:1,b:_0,track:narrow;path=a:2,b:_0,track:narrow;'\
+                      'path=a:0,b:_0;path=a:4,b:_0;',
+          },
+          'K1' => {
+            'count' => 1,
+            'color' => 'green',
+            'code' => 'label=K;city=revenue:30;path=a:0,b:_0,track:narrow;path=a:1,b:_0,track:narrow;path=a:2,b:_0;path=a:4,b:_0',
+          },
+          'B1' => {
+            'count' => 1,
+            'color' => 'green',
+            'code' => 'label=B;city=revenue:40;path=a:1,b:_0,track:narrow;path=a:2,b:_0,track:narrow;'\
+                      'path=a:3,b:_0;path=a:4,b:_0;',
+          },
+          'M1' => {
+            'count' => 1,
+            'color' => 'green',
+            'code' => 'label=M;city=revenue:30;path=a:4,b:_0;path=a:5,b:_0;path=a:2,b:_0,track:narrow;path=a:3,b:_0,track:narrow',
+          },
+          'T1' => {
+            'count' => 1,
+            'color' => 'green',
+            'code' => 'label=T;city=revenue:50,slots:2;path=a:0,b:_0;path=a:1,b:_0;path=a:4,b:_0;path=a:3,b:_0,track:narrow',
+          },
+          'Å1' => {
+            'count' => 1,
+            'color' => 'green',
+            'code' => 'label=Å;town=revenue:30;path=a:5,b:_0;path=a:1,b:_0,track:narrow;path=a:3,b:_0,track:narrow',
+          },
+          'LM1' => {
+            'count' => 1,
+            'color' => 'blue',
+            'code' => 'label=LM;junction;path=a:0,b:_0,terminal:1;junction;path=a:1,b:_1,terminal:2;'\
+                      'junction;path=a:2,b:_2,terminal:3;junction;path=a:3,b:_3,terminal:4;'\
+                      'junction;path=a:4,b:_4,terminal:5;junction;path=a:5,b:_5,terminal:6;'\
+                      'path=a:0,b:2,track:thin;path=a:0,b:3,track:thin;path=a:0,b:4,track:thin;' \
+                      'path=a:3,b:1,track:thin;path=a:3,b:5,track:thin;', \
```

Comment:
These LM tiles don't look to good. Would they be clearer if they only had the thin track on them? Do you need the thick track sections for anything (I haven't read the 18Norway rules yet)?

Reply:
Changed to thin lines only

Reply:
Thanks, I think these tiles look clearer now.